tests/automation/linked_device_user.spec.ts

```diff
@@ -22,273 +26,282 @@ import {
   waitForTextMessage,
 } from './utilities/utils';
 
-const windows: Array<Page> = [];
-test.beforeEach(beforeAllClean);
-
-test.afterEach(() => forceCloseAllWindows(windows));
-
-test('Link a device', async () => {
-  const [windowA] = await openApp(1); // not using sessionTest here as we need to close and reopen one of the window
-  const userA = await newUser(windowA, 'Alice');
-  const [windowB] = await linkedDevice(userA.recoveryPhrase); // not using sessionTest here as we need to close and reopen one of the window
-  await clickOnTestIdWithText(windowA, 'leftpane-primary-avatar');
-  // Verify Username
-  await waitForTestIdWithText(windowA, 'your-profile-name', userA.userName);
-  // Verify Session ID
-  await waitForTestIdWithText(windowA, 'your-session-id', userA.sessionid);
-  // exit profile module
-  await clickOnTestIdWithText(windowA, 'modal-close-button');
-  // You're almost finished isn't displayed
-  const errorDesc = 'Should not be found';
+sessionTestOneWindow('Link a device', async ([aliceWindow1]) => {
+  let aliceWindow2: Page | undefined;
   try {
-    const elemShouldNotBeFound = windowB.locator(
-      '[data-testid=reveal-recovery-phrase]',
+    const userA = await newUser(aliceWindow1, 'Alice');
+    aliceWindow2 = await linkedDevice(userA.recoveryPhrase); // not using fixture here as we want to check the behavior finely
+    await clickOnTestIdWithText(aliceWindow1, 'leftpane-primary-avatar');
+    // Verify Username
+    await waitForTestIdWithText(
+      aliceWindow1,
+      'your-profile-name',
+      userA.userName,
     );
-    if (elemShouldNotBeFound) {
-      console.error(
-        'Continue to save recovery phrase not found, excellent news',
+    // Verify Session ID
+    await waitForTestIdWithText(
+      aliceWindow1,
+      'your-session-id',
+      userA.sessionid,
+    );
+    // exit profile module
```

Comment:
module -> modal

Reply:
fixed (multiple times accross the source)